xiaomi.vacuum.gen1/audio_generator/Readme.md

```diff
@@ -0,0 +1,29 @@
+# Audio Generator for Xiaomi Vacuum Generation 1
+It is possible to create new language files which can be integrated into the rooted firmware. To generate the language files you can use the `generate_audio.py` script in this folder.
+
+It will read the content auf audio_xx.csv (whereby xx is a country language code like de_de or en_US) and use one of the supported engines to generate mp3 files for inclusion in the firmware. The language codes are the [ISO-639-1](https://en.wikipedia.org/wiki/ISO_639-1) format, so that the engine can use a voice with suitable pronounciation.
+
+You'll notice that each engine has its problems with mixed langauges (e.g. german with english words in it). A little testing is necessary to find a suitable pronounciation.
+
+# Requirements
+1. python3
+1. ffmpeg (to convert generated files into mp3)
```

Comment:
actually the gtts files are converted from mp3 to wav

Reply:
you're right. Found this out once I've tested the default way with gtts. I first thought I would need ffmpeg as well on mac os but fortunately I could write them as wav. I'll create another PR to fix this in the readme